builtin/providers/aws/resource_aws_rds_cluster.go

```diff
@@ -71,6 +71,12 @@ func resourceAwsRDSCluster() *schema.Resource {
 				Computed: true,
 			},
 
+			"storage_encrypted": &schema.Schema{
```

Comment:
The documentation update suggests that the default for this value is `false` - I think we need to actually have this in the schema as follows:

```
"storage_encrypted": &schema.Schema{
         Type:     schema.TypeBool,
 	Optional: true,
        Default: false,
 	ForceNew: true,
}
```

thoughts?